qt/README.md

```diff
@@ -0,0 +1,40 @@
+# toga-qt
+
+[![Python Versions](https://img.shields.io/pypi/pyversions/toga-qt.svg)](https://pypi.python.org/pypi/toga-qt)
+[![BSD-3-Clause License](https://img.shields.io/pypi/l/toga-qt.svg)](https://github.com/beeware/toga-qt/blob/main/LICENSE)
+[![Project status](https://img.shields.io/pypi/status/toga-qt.svg)](https://pypi.python.org/pypi/toga-qt)
+
+A Qt backend for the [Toga widget toolkit](https://beeware.org/toga).
+
+This package isn't much use by itself; it needs to be combined with
+[the core Toga library](https://pypi.python.org/pypi/toga-core).
+
+For platform requirements, see the Qt platform documentation (TODO).
+
+For more details, see the [Toga project on
+GitHub](https://github.com/beeware/toga).
+
+## Community
+
+Toga is part of the [BeeWare suite](https://beeware.org). You can talk
+to the community through:
+
+-   [@beeware@fosstodon.org on
+    Mastodon](https://fosstodon.org/@beeware)
+-   [Discord](https://beeware.org/bee/chat/)
+-   The Toga [GitHub Discussions
+    forum](https://github.com/beeware/toga/discussions)
+
+We foster a welcoming and respectful community as described in our
+[BeeWare Community Code of
+Conduct](https://beeware.org/community/behavior/).
+
+## Contributing
+
+If you experience problems with Toga, [log them on
+GitHub](https://github.com/beeware/toga/issues).
+
+If you'd like to contribute to Toga development, our [contribution
+guide](https://toga.readthedocs.io/en/latest/how-to/contribute/index.html)
```

Comment:
The .html link is no longer required, and we're using toga.beeware.org as the canonical docs link. We still working on getting that applied everywhere, but if we're adding new content, we can make it correct from the outset:
```suggestion
guide](https://toga.beeware.org/en/latest/how-to/contribute)
```